src/w3d/renderer/motchan.cpp

```diff
@@ -210,15 +228,15 @@ void TimeCodedMotionChannelClass::Get_Vector(float frame, float *setvec)
         int index2 = m_packetSize + index;
         unsigned int val = m_data[index2];
 
-        if (GameMath::Fast_Is_Float_Positive(val)) {
-            float f1 = (m_data[index] & 0x7FFFFFFF);
+        if (Get_Flag_From_Data(val)) {
```

Comment:
Hmm. Is this different from what it did earlier? Before it was converting integer to float to check the bit, now its checking the bit straight from unsigned integer.

Reply:
https://github.com/TheAssemblyArmada/Thyme/blob/56218d995d253f3f6b426ff7c0b83c1d481f0390/src/w3d/math/gamemath.h#L346
It was incorrectly using this function. As it's all inlined it's easy to get it mixed up.